python_modules/dagster/dagster/_cli/workspace/cli_target.py

```diff
@@ -496,8 +496,7 @@ def repository_click_options() -> Sequence[ClickOption]:
             "--location",
             "-l",
             help=(
-                "RepositoryLocation within the workspace, necessary if more than one location is"
-                " present."
+                "CodeLocation within the workspace, necessary if more than one location is present."
```

Comment:
Ditch 'workspace' too

Name of the code location, necessary if more than one location is present.